docs/cloud/tcld/namespace/update/search-attributes/add.md

```diff
@@ -0,0 +1,72 @@
+---
+id: add
+title: tcld namespace update search-attributes add
+sidebar_label: add
+description: How to add custom Search Attributes to a Namespace in Temporal Cloud using tcld.
+tags:
+  - reference
+  - tcld
+---
+
+The `tcld namespace update search-attributes add` command adds custom [Search Attributes](/docs/concepts/what-is-a-search-attribute) to a Namespace in Temporal Cloud.
+
+`tcld namespace update search-attributes add --search-attribute <value>`
+
+The following modifiers control the behavior of the command.
+
+### `--namespace`
+
+Specify a Namespace hosted on Temporal Cloud. If not specified, the value of the environment variable $TEMPORAL_CLOUD_NAMESPACE is used.
+
+Alias: `-n`
+
+**Example**
+
+```bash
+tcld namespace update search-attributes add --namespace <namespace_id> --search-attribute <value>
+```
+
+### `--request-id`
+
+Specify a request identifier to use for the asynchronous operation. If not specified, the server assigns a request identifier.
+
+Alias: `-r`
+
+**Example**
+
+```bash
+tcld namespace update search-attributes add --request-id <request_id> --search-attribute <value>
+```
+
+### `--resource-version`
+
+Specify a resource version (ETag) to update from. If not specified, the latest version is used.
+
+Alias: `-v`
+
+**Example**
+
+```bash
+tcld namespace update search-attributes add --resource-version <etag> --search-attribute <value>
+```
+
+### `--search-attribute`
+
+_Required modifier; can be specified more than once_
+
+Specify a custom Search Attribute in the form "_name_=_type_". Valid values for _type_ are as follows:
+
+- Bool
+- Datetime
+- Double
+- Int
+- Keyword
+- Text
+
+Alias: `--sa`
+
+**Example**
+
+```bash
+tcld namespace update search-attributes add --search-attribute "Category=Text" --search-attribute "LengthOverall=Double"
```

Comment:
nit: but these examples for the search attribute name confuses me (talking about "Category" and "LengthOverall" here).

Maybe something like "YourSearchAttributeName1=Text" and "YourSearchAttributeName2=Double"?